// MainMenu/index.js
module.exports = function ( index ) {
  var Model = require('backbone-model').Model;
  var Collect = require('backbone-collection').Collection;
  
  var place = require('Place');

  var MenuItem = Model.extend({
    defaults : {
      name      : "",
      href      : "",
      icon      : "",
      is_active : ""
    }
  });

  var MenuItemCollect = Collect.extend({
    model : MenuItem
  });
  
  return new MenuItemCollect([
      { name : "Главная"            , href : "/"            , icon : "fa-home"        , is_active : index == 'index' },
      { name : place.config.name    , href : "/place/"      , icon : "fa-folder"      , is_active : index == 'place' },
      { name : "Карты"              , href : "/maps/"       , icon : "fa-map-marker"  , is_active : index == 'maps' },
      { name : "Конверторы"         , href : "/converter/"  , icon : "fa-inbox"       , is_active : index == 'converter' },
      { name : "Пользователи"       , href : "/user/"       , icon : "fa-user"        , is_active : index == 'user' },
      { name : "Группы"             , href : "/group/"      , icon : "fa-group"       , is_active : index == 'group' },
      { name : "Роли"               , href : "/rule/"       , icon : "fa-eye-slash"   , is_active : index == 'rule' },
  ]);
};
